onboard/cv/main.cpp

```diff
@@ -125,13 +188,14 @@ int main() {
           arTags[0].id = -1;
         }
       } else { //one tag found
-        arTags[0].distance = depth_img.at<float>(tagPair.first.loc.y, tagPair.first.loc.x);
+        if(!isnan(depth_img.at<float>(tagPair.first.loc.y, tagPair.first.loc.x)))
+          arTags[0].distance = depth_img.at<float>(tagPair.first.loc.y, tagPair.first.loc.x);
```

Comment:
**Nit:** Even though there's only one line of code in the if-statement, I'd add brackets for readability. 

Also, in the future we should consider adding some method for searching around the tag's centerpoint for a non-`nan` depth value-- It wouldn't be too hard to do I think

Reply:
Absofruitly. Sounds like a good plan.